pgptool-gui/src/main/java/org/pgptool/gui/app/EntryPoint.java

```diff
@@ -99,7 +99,7 @@ public static void main(String[] args) {
 			splashScreenView = null;
 			entryPoint.startUp(args);
 			rootPmStatic = entryPoint.getRootPm();
-			log.debug("RootPM bean resolved");
```

Comment:
This is actually not a typo. "Bean" is a Spring term. Also can be referred as "Java Bean".

Reply:
yes, agreed.